app/controllers/LabelController.scala

```diff
@@ -57,6 +57,55 @@ class LabelController @Inject() (implicit val env: Environment[User, SessionAuth
     }
   }
 
+  /**
+   * Fetches the labels that a user has added in the current region they are working in.
+   *
+   * @param regionId Region id
+   * @return A list of labels
+   */
+  def getLabelsToResumeMission(regionId: Int) = UserAwareAction.async { implicit request =>
+    request.identity match {
+      case Some(user) =>
+        val labels: List[LabelTable.ResumeLabelMetadata] = LabelTable.getLabelsFromUserInRegion(regionId, user.userId)
+        val jsLabels: List[JsObject] = labels.map { label =>
+          Json.obj(
```

Comment:
Perhaps I'm missing something, but I don't see how I could use implicit conversion to convert a nested object into a flattened json object.

Reply:
I responded on Slack with a way to try and do this. If that doesn't work, then it's fine without the change. But worth trying first :)

Reply:
I'm thinking of just leaving it as is because I noticed there are some pretty large naming style inconsistencies in the Label.js. For example, there's a mix of snake case and camel case in the properties object of Label.js. However, the preexisting implicit conversion formats (like LabelWrites in LabelFormats.scala) fully use snake case. I think this is more of a style issue that is better fit for another issue (since we've al;ready delayed this PR for so long), where we can standardize the scala object -> JSON style better.

Reply:
yeah that's fine. yeah the mix of cases is a problem rn :sweat_smile: